app/src/main/java/com/rob729/quiethours/Adapter/ProfileListAdapter.kt

```diff
@@ -65,12 +70,60 @@ class ProfileListAdapter(
             binding.TxtImg.setText(item.name[0].toString())
             binding.TxtImg.avatarBackgroundColor = bgColors[item.colorIndex]
             binding.timeStamp.text = item.timeInstance
+            binding.pauseSwitch.isChecked = item.pauseSwitch
+            val current: Calendar = Calendar.getInstance()
+            val currentHour = current.get(Calendar.HOUR_OF_DAY)
+            val currentMinute = current.get(Calendar.MINUTE)
 
             binding.profileCard.setOnClickListener {
                 val args = Bundle()
                 args.putParcelable("Profile", item)
                 val dialog = DetailsFragment.newInstance(args)
-                dialog.show((parentView.context as FragmentActivity).supportFragmentManager, "DialogFragment")
+                dialog.show(
+                    (parentView.context as FragmentActivity).supportFragmentManager,
+                    "DialogFragment"
+                )
+            }
+            if (item.ehr == currentHour && item.emin == currentMinute
+            ) {
+                StoreSession.writeInt(AppConstants.BEGIN_STATUS, 0)
+                StoreSession.writeLong(AppConstants.ACTIVE_PROFILE_ID, 0)
+            }
+            if (item.shr == currentHour && item.smin == currentMinute
+            ) {
```

Comment:
why place ") {"  in a separate line?

Reply:
@rob729  I did not do it. I just used Ctrl+Alt+L to reformat the code. If it needs to be in one line, I will do it

Reply:
Okay, if it is due to Ctrl+Alt+L then let it remain unchanged.